source/developers-guide/plugin-system/index.md

```diff
@@ -386,6 +386,26 @@ class SwagControllerExample extends Plugin
 }
 ```
 
+### Controller auto-registration
+
+As of version `5.2.7` there is no need for a subscriber which registers your controllers. This will be automatically done by the Shopware core.
+Just place a controller in `SwagControllerExample/Controllers/Backend|Frontend|Widgets|Api` and you are done. 
```

Comment:
if you're using regex, use it the right way ;) `SwagControllerExample/Controllers/(Backend|Frontend|Widgets|Api)`